### ISSUE TITLE
table footer  total not refreshed

### ISSUE THREAD

**Issue by mvalletta** [grafana/grafana#57462]
**What happened**:
Table footer  total not refreshed

**What you expected to happen**:
After table table are refreshed , recalculate total in table footer

**How to reproduce it (as minimally and precisely as possible)**:
Create, wait 5 seconds or refresh dashboard, new random values are generated but total isn't refreshed

**Anything else we need to know?**:

**Environment**:
- Grafana version: 9.2.1
- Data source type & version: postgres 14.2
- OS Grafana is installed on: ubuntu 20.04
- User OS & Browser: ubuntu 20.04 chrome 116.0
- Grafana plugins:
- Others:

Follow panel from "More->Get help"

| Key | Value |
|--|--|
| Panel | table @ 9.2.1 |
| Grafana | 9.2.1 (34f7baebda) // Open Source |
<details><summary>Panel debug snapshot dashboard</summary>

```json
{
  "panels": [
    {
      "id": 2,
      "gridPos": {
        "h": 13,
        "w": 15,
        "x": 0,
        "y": 0
      },
      "type": "table",
      "title": "Reproduced with embedded data",
      "datasource": {
        "type": "grafana",
        "uid": "grafana"
      },
      "pluginVersion": "9.2.1",
      "description": "",
      "fieldConfig": {
        "defaults": {
          "custom": {
            "align": "auto",
            "displayMode": "auto",
            "inspect": false
          },
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "green",
                "value": null
              },
              {
                "color": "red",
                "value": 80
              }
            ]
          },
          "color": {
            "mode": "thresholds"
          }
        },
        "overrides": []
      },
      "options": {
        "showHeader": true,
        "footer": {
          "show": true,
          "reducer": [
            "sum"
          ],
          "fields": [
            "vakue"
          ]
        }
      },
      "targets": [
        {
          "refId": "A",
          "datasource": {
            "type": "grafana",
            "uid": "grafana"
          },
          "queryType": "snapshot",
          "snapshot": [
            {
              "schema": {
                "refId": "A",
                "meta": {
                  "executedQueryString": "SELECT now()-interval '3 minute' as time, random() * 100 as vakue\nunion \nSELECT now()-interval '2 minute' as time, random() * 100 as vakue\nunion \nSELECT now()-interval '1minute' as time, random() * 100 as vakue\nunion \nSELECT now() as time, random() * 100 as vakue\n"
                },
                "fields": [
                  {
                    "name": "Time",
                    "type": "time",
                    "typeInfo": {
                      "frame": "time.Time",
                      "nullable": true
                    },
                    "config": {}
                  },
                  {
                    "name": "vakue",
                    "type": "number",
                    "typeInfo": {
                      "frame": "float64",
                      "nullable": true
                    },
                    "config": {}
                  }
                ]
              },
              "data": {
                "values": [
                  [
                    1666367316672,
                    1666367136672,
                    1666367256672,
                    1666367196672
                  ],
                  [
                    30.629253467738238,
                    15.803936109913863,
                    57.5987730871919,
                    21.88480936806414
                  ]
                ]
              }
            }
          ]
        }
      ]
    },
    {
      "gridPos": {
        "h": 7,
        "w": 9,
        "x": 15,
        "y": 0
      },
      "id": 5,
      "options": {
        "content": "<table width=\"100%\">\n    <tr>\n      <th width=\"2%\">Panel</th>\n      <td >table @ 9.2.1</td>\n    </tr>\n    <tr>\n      <th>Queries</th>\n      <td>A[postgres]</td>\n    </tr>\n    \n    <tr><th>Data</th><td> 1 frames, 2 fields, 4 rows </td></tr>\n    \n    <tr>\n      <th>Grafana</th>\n      <td>9.2.1 (34f7baebda) // Open Source</td>\n    </tr>\n  </table>",
        "mode": "html"
      },
      "title": "Debug info",
      "type": "text"
    },
    {
      "id": 6,
      "title": "Original Panel JSON",
      "type": "text",
      "gridPos": {
        "h": 13,
        "w": 9,
        "x": 15,
        "y": 7
      },
      "options": {
        "content": "{\n  \"id\": 2,\n  \"gridPos\": {\n    \"h\": 9,\n    \"w\": 12,\n    \"x\": 0,\n    \"y\": 0\n  },\n  \"type\": \"table\",\n  \"title\": \"Panel Title\",\n  \"datasource\": {\n    \"type\": \"postgres\",\n    \"uid\": \"v-CiYCX7k\"\n  },\n  \"pluginVersion\": \"9.2.1\",\n  \"description\": \"\",\n  \"fieldConfig\": {\n    \"defaults\": {\n      \"custom\": {\n        \"align\": \"auto\",\n        \"displayMode\": \"auto\",\n        \"inspect\": false\n      },\n      \"mappings\": [],\n      \"thresholds\": {\n        \"mode\": \"absolute\",\n        \"steps\": [\n          {\n            \"color\": \"green\",\n            \"value\": null\n          },\n          {\n            \"color\": \"red\",\n            \"value\": 80\n          }\n        ]\n      },\n      \"color\": {\n        \"mode\": \"thresholds\"\n      }\n    },\n    \"overrides\": []\n  },\n  \"options\": {\n    \"showHeader\": true,\n    \"footer\": {\n      \"show\": true,\n      \"reducer\": [\n        \"sum\"\n      ],\n      \"fields\": [\n        \"vakue\"\n      ]\n    }\n  },\n  \"targets\": [\n    {\n      \"datasource\": {\n        \"type\": \"postgres\",\n        \"uid\": \"v-CiYCX7k\"\n      },\n      \"format\": \"time_series\",\n      \"group\": [],\n      \"metricColumn\": \"none\",\n      \"rawQuery\": true,\n      \"rawSql\": \"SELECT now()-interval '3 minute' as time, random() * 100 as vakue\\nunion \\nSELECT now()-interval '2 minute' as time, random() * 100 as vakue\\nunion \\nSELECT now()-interval '1minute' as time, random() * 100 as vakue\\nunion \\nSELECT now() as time, random() * 100 as vakue\\n\",\n      \"refId\": \"A\",\n      \"select\": [\n        [\n          {\n            \"params\": [\n              \"current_process_rate\"\n            ],\n            \"type\": \"column\"\n          }\n        ]\n      ],\n      \"table\": \"m3_campaign_campaign\",\n      \"timeColumn\": \"started_tstamp\",\n      \"timeColumnType\": \"timestamp\",\n      \"where\": [\n        {\n          \"name\": \"$__timeFilter\",\n          \"params\": [],\n          \"type\": \"macro\"\n        }\n      ]\n    }\n  ]\n}",
        "mode": "code",
        "code": {
          "language": "json",
          "showLineNumbers": true,
          "showMiniMap": true
        }
      }
    },
    {
      "id": 3,
      "title": "Data from panel above",
      "type": "table",
      "datasource": {
        "type": "datasource",
        "uid": "-- Dashboard --"
      },
      "gridPos": {
        "h": 7,
        "w": 15,
        "x": 0,
        "y": 13
      },
      "options": {
        "showTypeIcons": true
      },
      "targets": [
        {
          "datasource": {
            "type": "datasource",
            "uid": "-- Dashboard --"
          },
          "panelId": 2,
          "withTransforms": true,
          "refId": "A"
        }
      ]
    }
  ],
  "schemaVersion": 37,
  "title": "Debug: Panel Title // 2022-10-21 17:48:40",
  "tags": [
    "debug",
    "debug-table"
  ],
  "time": {
    "from": "2022-10-21T09:48:40.805Z",
    "to": "2022-10-21T15:48:40.805Z"
  }
}
```
</details>

**Comment by sydevinfra** [comment]
Same issue.

Grafana version: 9.2.1
Data source type & version: elasticsearch 8.4.3
OS Grafana is installed on: RHEL 8.2
User OS & Browser: Windows 10, Chrome 106.0
Grafana plugins: N/A
Others: N/A

**Comment by codeincarnate** [comment]
Thanks for the report. This was sadly a regression from another change but has already been fixed with #57518. This should be going out with the next release 😄.
